server.ts

```diff
@@ -126,6 +126,36 @@ async function prepareDataForDB(data, properties) {
     return out
 }
 
+async function initHelloWorldGit() {
+
+    const connection = await db.connect()
+    
+    const helloWorldGit = await connection.getRepository(Git).findOne('hello_world')
+    
+    if (!helloWorldGit) {
+    
+    const git = {
+    
+    id: 'hello_world',
+    
+    address: `https://github.com/cybergis/cybergis-compute-hello-world.git`,
+    
```

Comment:
The code looks great. Though, could you work on the indentations and remove some extra likes in the code? It seems not well formatted. Overall, code looks good to me.